testsuite/domain/src/test/resources/host-configs/admin-only-discovery.xml

```diff
@@ -27,7 +27,7 @@
         <!-- Remote domain controller configuration with a host and port -->
         <remote authentication-context="secondaryHostAContext" ignore-unused-configuration="true" admin-only-policy="${jboss.test.admin-only-policy}">
             <discovery-options>
-                <static-discovery name="primary" host="${jboss.test.host.primary.address}" port="9999"/>
+                <static-discovery name="primary" protocol="remote" host="${jboss.test.host.primary.address}" port="9999"/>
```

Comment:
These additions look problematic, if we need to make configuration changes in test cases as a result of work in the management model this indicates that these changes will potentially break existing  user configurations that don't have these attributes defined.